base/main.cpp

```diff
@@ -209,12 +208,14 @@ static Common::Error runGame(const EnginePlugin *plugin, OSystem &system, const
 	Common::String caption(ConfMan.get("description"));
 
 	if (caption.empty()) {
-		caption = EngineMan.findGame(ConfMan.get("gameid")).description();
+	    caption = EngineMan.findGame(ConfMan.get("gameid")).description();
```

Comment:
We use tabs and not spaces for indentation.

Actually it doesn't look like any of the changes in this file do anything, so I would suggest to just drop all these changes from your commit.